config/application.rb

```diff
@@ -42,5 +42,7 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.middleware.insert_after ActiveRecord::Migration::CheckPending, ActionDispatch::Cookies
```

Comment:
What are these for?

Reply:
This is required by omniauth for session management.
https://github.com/omniauth/omniauth#integrating-omniauth-into-your-rails-api